test/utils/helpers.test.ts

```diff
@@ -34,3 +34,35 @@ describe('isTimestampMoreRecentThan', () => {
     });
   });
 });
+
+describe('.formatWeiToGwei', () => {
+  describe('when wei is lesser than 100000 (1e5)', () => {
+    it('returns zero', () => {
+      expect(formatWeiToGwei(1)).to.equal(0);
+      expect(formatWeiToGwei(1e1)).to.equal(0);
+      expect(formatWeiToGwei(1e2)).to.equal(0);
+      expect(formatWeiToGwei(1e3)).to.equal(0);
+      expect(formatWeiToGwei(1e4)).to.equal(0);
+    });
+  });
+
```

Comment:
Maybe add a describe here:
`describe('when wei is bigger than 10000(1e4)', () => {`

Reply:
I've added a new describe `when wei is equal or greater than 100000 (1e5)`.